internal/i18n/en_translations.go

```diff
@@ -133,7 +133,7 @@ var (
 	MsgUnknownDataExchangePlugin   = ffm("FF10213", "Unknown Data Exchange plugin '%s'")
 	MsgParentIdentityNotFound      = ffm("FF10214", "Organization with identity '%s' not found in identity chain for %s '%s'")
 	MsgInvalidSigningIdentity      = ffm("FF10215", "Invalid signing identity")
-	MsgNodeAndOrgIDMustBeSet       = ffm("FF10216", "node.name, org.name and org.identity must be configured first", 409)
+	MsgNodeAndOrgIDMustBeSet       = ffm("FF10216", "node.name, org.name and org.key must be configured first", 409)
```

Comment:
Unrelated to the chart changes, but this error message confused me a bit since this is whats displayed when neither `org.key` nor `org.identity` are set (which was happening w/ my chart installation due to a bug on my end). It should references the new config key instead rather than the deprecated one.